go.mod

```diff
@@ -0,0 +1,32 @@
+module github.com/jetstack/kube-oidc-proxy
+
+go 1.12
+
+require (
+	github.com/Masterminds/semver v1.4.2
+	github.com/golang/mock v0.0.0-20160127222235-bd3c8e81be01
+	github.com/heptiolabs/healthcheck v0.0.0-20180807145615-6ff867650f40
+	github.com/sirupsen/logrus v1.4.2
+	github.com/spf13/cobra v0.0.5
+	github.com/spf13/pflag v1.0.3
+	gopkg.in/square/go-jose.v2 v2.3.1
+	k8s.io/api v0.0.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/apiserver v0.0.0-20190721103406-1e59c150c171
+	k8s.io/cli-runtime v0.0.0
+	k8s.io/client-go v11.0.0+incompatible
+	k8s.io/component-base v0.0.0
+	k8s.io/klog v0.3.3
+	sigs.k8s.io/kind v0.0.0-00010101000000-000000000000
```

Comment:
I've seen this happen before with kind and I'm not 100% certain why.. I've you run `./hack/pin-dependency.sh sigs.k8s.io/kind v0.4.0`, I think it should update this string to be correct too. I don't think it's that important to do though given the `replace` directive + go.sum file.

Reply:
Changed it to 0.4.0 now. Just a cosmetic change in mod, no vendor changes